lib/kafo/execution_environment.rb

```diff
@@ -19,6 +19,15 @@ def directory
       end
     end
 
+    def reportdir
+      @reportdir ||= File.join(directory, 'reports')
+    end
+
+    def reports
+      # Reports are stored in $reportdir/$certname/$report
+      Dir.glob(File.join(reportdir, '*', '*.*')).sort_by { |path| File.mtime(path) }
+    end
```

Comment:
Do reports get cleaned up automatically over time?

Reply:
It is within the execution environment, which is all in a temporary directory that gets cleaned up at the exit of the program:
https://github.com/theforeman/kafo/blob/78f565eeaf6dfb405a9dafcfc52cde4d82e9f5a0/lib/kafo/kafo_configure.rb#L486-L487